docs/ide/tutorial-csharp-console.md

```diff
@@ -132,7 +128,7 @@ If you don't see the **Console App (.NET Core)** project template, you can get i
 
    ![Code editor showing the C# Console Calculator](../ide/media/csharp-console-calculator-code.png)
 
-1. Choose **Calculator** to run your program, or press **F5**.
+1. Press **Calculator** or **F5** to run your program.
```

Comment:
We use "choose" in instances where someone would "click" something, and "press" where someone would physically press a key. (We use "choose" instead of "click" because some people might use a device with touch controls enabled.) Respectfully, I decline this edit.